app/mailers/user_mailer.rb

```diff
@@ -448,4 +448,45 @@ def verify_email_after_120_hours(user_id)
 		end
 	end
 
+	def export_all_subjective_responses(email)
+		consultation_list = Consultation.includes(:responses).select { |con| con.response_rounds.last.questions.count == 0}
```

Comment:
@shrudhinvraj  - How many consultations will be there?

Reply:
Do you have the count?

Reply:
@manikandan0603  there will be 313 consultations which have subjective responses